worker/src/PayloadChannel/PayloadChannelSocket.cpp

```diff
@@ -196,94 +224,6 @@ namespace PayloadChannel
 		  this->uvReadHandle,
 		  this->payloadChannelReadCtx);
 
-		if (free)
```

Comment:
And the same here, you just dropped Rust support, while it had no effect on TypeScript